code/game/area/Space Station 13 areas.dm

```diff
@@ -829,6 +829,10 @@ proc/process_adminbus_teleport_locs()
 	name = "Abandoned Diner"
 	icon_state = "dinerderelict"
 
+/area/derelictparts/incinerator

```

Comment:
Move to maps/defficiency/areas.dm so all the map shit is in one bundle.  Use ancientstation.dm as an example on how to properly include the *.dm's.